admin/InitDb.pl

```diff
@@ -228,14 +228,9 @@ sub Create
     $system_sql->auto_commit;
     $system_sql->do("ALTER DATABASE $dbname SET timezone TO 'UTC'");
 
-    # You can do this via CREATE FUNCTION, CREATE LANGUAGE; but using
-    # "createlang" is simpler :-)
-    my $sys_db = Databases->get($sysname);
-    my $sys_in_thisdb = $sys_db->meta->clone_object($sys_db, database => $dbname);
-    my @opts = $sys_in_thisdb->shell_args;
-    splice(@opts, -1, 0, "-d");
-    $ENV{"PGPASSWORD"} = $sys_db->password;
-    system "createlang", @opts, "plpgsql";
+    $system_sql->auto_commit;
+    $system_sql->do("CREATE OR REPLACE LANGUAGE plpgsql");
```

Comment:
Wait, is this even needed? My copy of PG 9.5 already has the plpgsql language in the template1 database (it can't be dropped, because "extension plpgsql requires it"), which means the MB database already inherits the language.